product_state/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============
+Product State
+=============
+
+In version 10.0 Odoo removes the state field from product. This module re-introduces it and allows simple product life cycle.
```

Comment:
IMO you should get rid of the first part and have something like : 'This module introduces the state field on product template and allows simple product life cycle.'